apps/frontend/src/renderer/components/project-settings/IntegrationSettings.tsx

```
@@ -481,6 +484,81 @@ export function IntegrationSettings({
           </div>
         )}
       </section>
+
+      <Separator />
+
+      {/* Git Provider Section - NEW */}
+      <section className="space-y-3">
+        <button
+          onClick={() => setGitProviderExpanded(!gitProviderExpanded)}
+          className="w-full flex items-center justify-between text-sm font-semibold text-foreground hover:text-foreground/80"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)